### PR TITLE
Change the order of requests and form of binding

### PR DESCRIPTION
**Pull request by martijnjanssen**
This makes timing of loading the data less flow bound, which fixes the total counter missing.

### CODE REVIEW

File: src/lancie-admin-progress/lancie-admin-progress.html [hunk 1]
```diff
@@ -78,17 +77,16 @@
...
       for (var c in this.consumptions) {
-        this.addProgressMeter(c, this.consumptions[c], this.tickets);
+        this.addProgressMeter(c, this.consumptions[c]);
       }
     }
   }
 
   // Adds a new progressMeter with the Polymer push function to notify
-  addProgressMeter(name, count, total) {
+  addProgressMeter(name, count) {
```
Comment (elarb) [inline src/lancie-admin-progress/lancie-admin-progress.html]: I think that its also useful to check whether a progress meter with the same name doesn't exist already? And remove it if it does exist. Also, I think that it would be even better if the logic for the progress status was done in the API and that one request was sufficient, but that would be for another PR

File: src/lancie-admin-progress/lancie-admin-progress.html [hunk 2]
```diff
@@ -47,14 +47,13 @@
...
       this.tickets = request.response.length;
-      this.$.ajaxRfid.generateRequest();
     }
   }
 
   // Retrieves all people checked in
   onRfid(event, request) {
     if (request.succeeded) {
-      this.addProgressMeter('CheckIn', request.response.length, this.tickets);
+      this.addProgressMeter('CheckIn', request.response.length);
```
Comment (elarb) [inline src/lancie-admin-progress/lancie-admin-progress.html]: Check-in instead of CheckIn maybe?

**Review by elarb — COMMENTED** [review]
A few comments

File: src/lancie-admin-progress/lancie-admin-progress.html (hunk 1 above)
Comment (martijnjanssen) [reply src/lancie-admin-progress/lancie-admin-progress.html]: I agree for the api part, it would be better to track it there. I'll add the check though.

**Review by elarb — APPROVED** [review]
Didn't test it after the last commit, but looks good
